### ISSUE TITLE
Hey

### ISSUE THREAD

**Issue by nrobs** [haoict/twitch-no-ads#2]
The Adblock is not working for me :(

**Comment by shtuseriff** [comment]
> The Adblock is not working for me :(

It’s not working for anyone right now. Look at the other issue that’s posted on this repo... someone there had a recommendation you could try (VPN)

**Comment by abraham8p** [comment]
> The Adblock is not working for me :(

yep, use one of the vpns I suggested in the other Issue I posted or just wait for a update :/
